src/workerd/api/urlpattern-test.js

```diff
@@ -1,518 +0,0 @@
-// Copyright (c) 2023 Cloudflare, Inc.
```

Comment:
is there a reason we don't like these tests anymore? they would give some additional safety too.

Reply:
These tests still exist. They were moved into `tests/url-tests` to keep all of the URL-standard related tests together.